### DISCUSSION TITLE
CouchDB

### DISCUSSION THREAD

**Discussion by adamemrts** [coollabsio/coolify#3889] [Service Template Requests]
CouchDB as database

**Comment by eterps** [comment]
When you search for `couchdb` in the documentation you get:

> One-click MongoDB, MySQL, PostgreSQL, CouchDB, RedisDB instances ready to use, locally or over the internet.

Under [features](https://docs.coollabs.io/coolify/v3/introduction#features).

But apparently this is not true? Or only in a preview release? It's not entirely clear.
